src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -26,7 +26,7 @@ public sealed class DefaultHttpContext : HttpContext
 
         private DefaultHttpRequest _request;
         private DefaultHttpResponse _response;
```

Comment:
Nit: readonly

Reply:
Can't, they get re-initialized in Initialize